chapters/VK_EXT_debug_utils.txt

```diff
@@ -66,17 +74,6 @@ calling fname:vkSetDebugUtilsObjectNameEXT again with a new string.
 If pname:pObjectName is an empty string, then any previously set name is
 removed.
 
-.Valid Usage
-****
-  * [[VUID-VkDebugUtilsObjectNameInfoEXT-objectType-01905]]
-    pname:objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
-  * [[VUID-VkDebugUtilsObjectNameInfoEXT-objectHandle-01906]]
-    pname:objectHandle must: not be dlink:VK_NULL_HANDLE
```

Comment:
Might still make sense to ban Unknown+Null. The app  an just pass 0 object count for that case.

Reply:
How about:

```
  * If pname:objectType is ename:VK_OBJECT_TYPE_UNKNOWN,
    pname:objectHandle must: not be dlink:VK_NULL_HANDLE
```

Reply:
Actually that would ban exactly the case we wanted to allow (i.e. allow NULL_HANDLE object from `vkSubmitDebugUtilsMessageEXT`)?

Reply:
I think we agreed that we do not want `VK_OBJECT_TYPE_UNKNOWN` cases with valid handle, or did you reconsider?

Reply:
No, I'm still leaning towards allowing it because the validation layers could return it.  If it's in a valid usage statement on the object, people might think the layers are behaving incorrectly.

In the case of an extension not supported by the layers, that adds a new type, it may call a supported command with the new type value which won't be known resulting in an error message with `UNKNOWN`.